Flick/Cells/ListSummaryCollectionViewCell.swift

```diff
@@ -204,6 +171,45 @@ class ListSummaryCollectionViewCell: UICollectionViewCell {
         }
     }
 
+    private func setupCollaborators(collaborators: [UserProfile]) {
+        collaborateLabel.text = collaborators.count <= 1 ? Constants.Collaboration.onlyICanEdit : Constants.Collaboration.numCanEdit(num: collaborators.count)
+        collaboratorsPreviewView.users = collaborators
+
+        let listInfoHeight = 20
+        let numCollaborators = min(collaborators.count, 7)
+        let fullCollaboratorsWidth = numCollaborators * 20
+        let overlapCollaboratorsWidth = (numCollaborators - 1) * collaboratorsCellSpacing * -1
+        let collaboratorsPreviewWidth = fullCollaboratorsWidth - overlapCollaboratorsWidth
```

Comment:
I wonder if we can make this a function inside of UserPreviewView? I think we talked about this before too but I guess it makes sense for the component itself to have the function to calculate its width. We can do this in this PR or make it another smaller task? cause it would require changing the other codes

Reply:
yea there's #40 
i think i could add that to my next PR